Formula/augustus.rb

```diff
@@ -0,0 +1,28 @@
+class Augustus < Formula
+  desc "Predict genes in eukaryotic genomic sequences"
+  homepage "http://bioinf.uni-greifswald.de/augustus/"
+  url "http://bioinf.uni-greifswald.de/augustus/binaries/augustus-3.3.tar.gz"
+  sha256 "f6470931e835d8da16288e501f95287df0f71f6bd6b6c8b49c3cf33c1e7f18e5"
+
+  depends_on "boost"
+
+  def install
+    # Prevent symlinking into /usr/local/bin/
+    inreplace "Makefile", %r{ln -sf.*/usr/local/bin/}, "#ln -sf"
```

Comment:
it would be good to request some way of turning this off without using string replacements. It makes no sense to me why it would do this with a hard-coded path if it's allowing us to set INSTALLDIR